5.sql

```diff
@@ -0,0 +1,30 @@
+USE banking_db
+GO
+
+CREATE OR ALTER PROCEDURE SocialSubsidy (@inputStatus NVARCHAR(30) = '���������', @inputMoneyIncrease MONEY = 10)
+AS
+BEGIN
+	
+	SELECT Accounts.UserName, Accounts.AccountMoney

```

Comment:
It is better to make checks (Select queries) before calling the procedure and after it

Select
Exec
Select